runtime/src/types.rs

```diff
@@ -103,24 +103,28 @@ mod metadata_aliases {
 
     pub type InterBtcHeader = <InterBtcRuntime as Config>::Header;
 
-    pub type InterBtcIssueRequest =
-        metadata::runtime_types::interbtc_primitives::issue::IssueRequest<AccountId, BlockNumber, Balance, CurrencyId>;
+    pub type InterBtcIssueRequest = metadata::runtime_types::interbtc_primitives::issue::IssueRequest<
+        Static<AccountId>,
+        BlockNumber,
+        Balance,
+        Static<CurrencyId>,
+    >;
     pub use metadata::runtime_types::interbtc_primitives::issue::IssueRequestStatus;
     pub type InterBtcRedeemRequest = metadata::runtime_types::interbtc_primitives::redeem::RedeemRequest<
-        AccountId,
+        Static<AccountId>,
         BlockNumber,
         Balance,
-        CurrencyId,
+        Static<CurrencyId>,
     >;
     pub use metadata::runtime_types::interbtc_primitives::{
         redeem::RedeemRequestStatus, replace::ReplaceRequestStatus,
     };
 
     pub type InterBtcReplaceRequest = metadata::runtime_types::interbtc_primitives::replace::ReplaceRequest<
-        AccountId,
+        Static<AccountId>,
         BlockNumber,
         Balance,
-        CurrencyId,
+        Static<CurrencyId>,
     >;
     pub type VaultId = metadata::runtime_types::interbtc_primitives::VaultId<AccountId, CurrencyId>;
```

Comment:
@sander2 and @gregdhill can i make the 
` pub type VaultId = metadata::runtime_types::interbtc_primitives::VaultId<Static<AccountId>, Static<CurrencyId>>;
` this will resolve the rest of the todos. But my question is will this cause issues in any of the upstream dependency for this type.  This is required because of the new wrapping done to the types [here](https://github.com/interlay/interbtc-clients/blob/7d7a9fe2234fd8aa3bb659ffa17ea00151b4df28/runtime/src/lib.rs#L201) and [here](https://github.com/interlay/interbtc-clients/blob/7d7a9fe2234fd8aa3bb659ffa17ea00151b4df28/runtime/src/lib.rs#L205). 

Reference: [Issue raised on subxt](https://github.com/paritytech/subxt/issues/1039)

Reply:
Happy to go with either solution as long as the tests pass. We can do a final review once the pipeline is green.